spec/v1/envelope.md

```diff
@@ -57,10 +80,16 @@ Example media types for single DSSE-signed attestation predicates include:
 -   SPDX: `application/vnd.in-toto.spdx+dsse`
 -   VSA: `application/vnd.in-toto.vsa+dsse`
 
+[^1]: There is an [ongoing discussion](https://github.com/sigstore/sig-clients/issues/9) about supporting [DSSE Signature Extensions](https://github.com/secure-systems-lab/dsse/blob/devel/envelope.md#signature-extensions-experimental) to extend the current features of Sigstore Bundles.
+
 [Bundle]: bundle.md
-[DSSE v1.0]: https://github.com/secure-systems-lab/dsse/blob/v1.0.0/envelope.md
+[COSE\_Sign]: https://datatracker.ietf.org/doc/html/rfc8152#section-4.1
```

Comment:
Is the `\` intended?

Reply:
I added the `\` because I was seeing some markdown rendering weirdness without it. But there is another place where I need to remove it :)

Reply:
turns out the `\` aren't needed after all. I removed them.